### ISSUE TITLE
Update screenshots

### ISSUE THREAD

**Issue by hugolpz** [lingua-libre/SignIt#46]
Icons have been recently updated (#39's phase 1 & phase 2).
The documentation can now be updated by redoing the screenshoots and spreading those to the relevant places.

#### Phase 3
The task below are good first issues.
- [ ] recreate documentation screenshots https://github.com/lingua-libre/SignIt/tree/master/doc
- [ ] update commons screenshots https://commons.wikimedia.org/wiki/Category:Lingua_Libre_SignIt
  - [ ] meta https://meta.wikimedia.org/wiki/Lingua_Libre/SignIt
  - [ ] wikidata https://www.wikidata.org/wiki/Q113354157

#### Phase4
@hugolpz 
- [ ] redeploy on firefox addon store https://addons.mozilla.org/en-US/firefox/addon/lingua-libre-signit/

**Comment by JatinSainiOO7** [comment]
Hi  @hugolpz ,
### Can I  work on this issue
Thankyou

**Comment by hugolpz** [comment]
@shashankiitbhu, yes you can. It's relatively easy : you just have to install the dev version on your computer and redo the screenshots. The aim of this issue is mostly to let you at the same time try SignIt on FireFox and help the project by updating its screenshot. Simple but helpful.

@shashankiitbhu @JatinSainiOO7, I will open a new issue to discuss with you were to get involved if you wouldlike to.

**Comment by Nama21yo** [comment]
Hi @hugolpz,
Is anyone already working on this, or can I take it on? If there are a specific guidelines or things should keep in mind, please let me know.
Looking forward to contributing!

**Comment by hugolpz** [comment]
Hello @Nama21yo, the codebase and UI are nearly stable but may change in coming 2 months. So in order to save good will volunteer's time, may I rather contact you back later this Automn for that ?

**Comment by Nama21yo** [comment]
Ok Thanks for your response. I understand the timing and will patiently wait until then.
